Readme docs updates: pip, composer & sudo build.sh

JasonHoku opened the pull request:
pip --> pip3 for more global install stability.

 composer docs added for clarity 

sudo build.sh ( this is necessary with the default docker setup )

:)

JasonHoku commented:
Any idea why my VSCodium edit of this would yield these result?

Buuntu commented on README.md:
```diff
@@ -48,31 +48,34 @@ modern stack.
-    - [PrivateRoute](#privateroute)
-- [Deployment](#deployment)
-- [Contributing](#contributing)
+- [FastAPI + React · !build [![license](https://img.shields.io/github/license/peaceiris/actions-gh-pages.svg)](LICENSE) [![Dependabot Status](https://img.shields.io/badge/Dependabot-active-brightgreen.svg)](https://dependabot.com)](#fastapi--react--img-srchttpsgithubcombuuntufastapi-reactworkflowsbuildbadgesvg-altbuild-img-srchttpsimgshieldsiogithublicensepeaceirisactions-gh-pagessvg-altlicense-i ...
```
This, features, and table of contents should not be a part of the TOC.

Buuntu commented on README.md:
```diff
@@ -122,11 +132,11 @@ You will need to put in a few variables and it will create a project directory
 ## Develop
 
-Change into your project directory and run:
+Change into your project directory and run as sudo priv (required for docker):
```
Run with elevated privileges is better.

Buuntu commented on README.md:
````diff
@@ -122,11 +132,11 @@ You will need to put in a few variables and it will create a project directory
 ```bash
-chmod +x scripts/build.sh
-./scripts/build.sh
+sudo chmod +x scripts/build.sh
````
I don't think you need sudo for this, you might for docker-compose is all.

JasonHoku commented:
Okay, I have corrected the strange formatting caused a MD formatted for VS Code and removed the sudo for the build.sh

Still got the prettier failed error, is that okay?

Again, thank you so much for working with me on this.

Buuntu commented:
> Okay, I have corrected the strange formatting caused a MD formatter for VS Code and removed the sudo for the build.sh
> 
> Still got the prettier failed error, is that okay?
> 
> Again, thank you so much for working with me on this.

I think you need to setup Prettier in VSCode, it may not be running on the README.
